pkg/cloud/api/mutable_resource.go

```diff
@@ -168,6 +186,10 @@ type mutableResource[GA any, Alpha any, Beta any] struct {
 }
 
 func (u *mutableResource[GA, Alpha, Beta]) CheckSchema() error {
+	if isPlaceholderType(u.ga) {
+		return fmt.Errorf("GA cannot be PlaceholderType")
```

Comment:
Type errors should always print out type and any other information that will be useful for debugging.

fmt.Errorf("GA cannot be PlaceholderType (type is %T)", u)

Reply:
but time is included in error :) 
maybe sth like fmt.Errorf("GA has unsupported type (type is %T)", u)

Reply:
Ok -- can you make the change in a follow up PR? I am going to merge this.